recipes/sophus/build.sh

```diff
@@ -0,0 +1,10 @@
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -DCMAKE_PREFIX_PATH=${PREFIX} \
+      -DBUILD_SOPHUS_EXAMPLES=OFF \
+      -DBUILD_SOPHUS_TESTS=OFF \
+      -DSOPHUS_USE_BASIC_LOGGING=ON \
+      ..
+make -j${CPU_COUNT}
+make install
```

Comment:
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
      -DCMAKE_PREFIX_PATH=${PREFIX} \
      -DBUILD_SOPHUS_EXAMPLES=OFF \
      -DBUILD_SOPHUS_TESTS=ON \
      -DSOPHUS_USE_BASIC_LOGGING=ON \
      ..
make -j${CPU_COUNT}
make test
make install
```
You should build and run at least one test to check that the conda-forge compilers are compatible with the source and compiler args.